core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -791,7 +791,7 @@ class WorldScreen(val gameInfo: GameInfo, val viewingCiv:CivilizationInfo) : Bas
                 }
 
             !viewingCiv.hasMovedAutomatedUnits && viewingCiv.getCivUnits()
-                .any { it.isMoving() || it.isAutomated() || it.isExploring() } ->
+                .any { it.currentMovement >= 0.1f && (it.isMoving() || it.isAutomated() || it.isExploring()) } ->
```

Comment:
I personally prefer `currentMovement > 0` or `currentMovement >= Constants.MinimumMovementEpsilon` here, but I suppose this has been merged for a few days already